events/src/main/java/tech/pegasys/artemis/events/EventChannel.java

```diff
@@ -134,11 +134,16 @@ public T getPublisher() {
     return publisher;
   }
 
-  public void subscribe(T listener) {
+  public void subscribe(final T listener) {
+    subscribeMultithreaded(listener, 1);
+  }
+
+  public void subscribeMultithreaded(final T listener, final int numberOfThreads) {
```

Comment:
I've added some java doc.  I'm not too concerned about the fact that the number of threads is ignored for synchronous channels because we should only use those for testing anyway.

I'm also more concerned that we'll accidentally make something multithreaded than that it will be single threaded.  So have stuck with `subscribeMultithreaded` as the name so it's really clear.